comfy/cli_args.py

```diff
@@ -1,6 +1,8 @@
 import argparse
 import enum
 import comfy.options
+from app.frontend_management import FrontendManager
```

Comment:
comfy/cli_args.py should not import this file.

Reply:
Things inside the comfy folder should not depend on things outside of it.

Reply:
Inlined.